src/libNode/MicroBlockProcessing.cpp

```diff
@@ -502,6 +698,20 @@ bool Node::RunConsensusOnMicroBlockWhenShardLeader()
               "I am shard leader. Creating microblock for epoch"
                   << m_mediator.m_currentEpochNum);
 
+    bool isVacuousEpoch
+        = (m_consensusID >= (NUM_FINAL_BLOCK_PER_POW - NUM_VACUOUS_EPOCHS));
+    if (!isVacuousEpoch)
```

Comment:
Needless to use extra variable (isVacuousEpoch) to remember the conditional checking result

Reply:
There is too many similar code like this, let's do a refactoring for all of them in a seperate pr